frontend/app/components/Session_/Inspector/ElementView.tsx

```diff
@@ -88,6 +91,7 @@ export default function ElementView({
 	selectedElement,
 	setSelectedElement,
 	onHover,
+	className='',
```

Comment:
Please, use `className?` declaration in this situation instead, there is no point in default value